docs/spelling_wordlist.txt

```diff
@@ -88,6 +89,7 @@ interoperability
 io
 ipynb
 iteratively
+IWorkflowProvider
```

Comment:
It's weird that you had to add this (and some others, like `str` and `cls`), since we don't need them in the rp docs.

Reply:
maybe this was an issue with `nitpick_ignore` not working... I'll check

Reply:
indeed those were not needed. Probably triggered before I setup the google docstring style.